<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewIntervalForm</class>
 <widget class="QWidget" name="NewIntervalForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	text-align: center;
	font: 57 16pt &quot;Sono Medium&quot;;
}

QCheckBox {
	text-align: center;
	font: 57 16pt &quot;Sono Medium&quot;;
}

QCheckBox:focus {
    border: 2px solid #0057D9;
    background-color: #E6F4FF;
	border-radius: 5px;
}

QRadioButton {
	text-align: center;
	font: 57 16pt &quot;Sono Medium&quot;;
}

QRadioButton:focus {
    border: 2px solid #0057D9;
    background-color: #E6F4FF;
	border-radius: 5px;
}

QWidget {
    background-color: rgb(208, 225, 249);
}

QPushButton {
    background-color: rgb(77, 100, 141);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton:focus {
    background-color: rgb(40, 54, 85);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton[class=&quot;CustomSpinboxButton&quot;] {
    background-color: rgb(77, 100, 141);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton[class=&quot;CustomSpinboxButtonPressed&quot;] {
    background-color: rgb(40, 54, 85);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QDoubleSpinBox {
	text-align: centrel;
    border: none;
	border-radius: 5px;
    background-color: #E6F4FF;
	font: 57 16pt &quot;Sono Medium&quot;;
}

QDoubleSpinBox:focus {
    border: 2px solid #0057D9;
    background-color: #E6F4FF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Проект:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Скважина:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="project_name_label">
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="well_name_label">
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="back_button">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item row="1" column="1">
      <widget class="QPushButton" name="interval_from_decrease_button">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="interval_from_doubleSpinBox">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>99999.990000000005239</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="interval_from_increase_button">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>От:</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>До:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="interval_to_increase_button">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="interval_to_decrease_button">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="interval_to_doubleSpinBox">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>99999.990000000005239</double>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QCheckBox" name="is_marked_checkBox">
       <property name="text">
        <string>С разметкой</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="wet_radioButton">
       <property name="text">
        <string>Мокрый</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dry_radioButton">
       <property name="text">
        <string>Сухой</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="create_new_interval_button">
     <property name="text">
      <string>Создать интервал</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
